.github/workflows/integration-tests.yml

```diff
@@ -201,6 +212,7 @@ jobs:
     strategy:
       matrix:
         key: [8, 16]
+        auth: [None, Basic]
```

Comment:
There is no need to run aliens test with auth. gRPC auth was already tested within put-get-exist test

Reply:
But you need to add auth for `bobt` test, because `bobt` works through REST API